build.xml

```diff
@@ -145,6 +147,20 @@
         <fail if="tests.failed" message="There were failed unit tests"/>
     </target>
 
+    <target name="test" depends="compile, set_excluded_test_groups" description="Run unit tests">
+        <antcall target="run-test">
+            <param name="excludedTestGroups" value="${excludedTestGroups}, sra"/>
+        </antcall>
+
+    </target>
+    <target name="sra-test" depends="compile, set_excluded_test_groups" description="Run SRA unit tests">
+        <antcall target="run-test">
+            <param name="includedTestGroups" value="sra"/>
+            <param name="excludedTestGroups" value="${excludedTestGroups}"/>
```

Comment:
The params in an `antcall` will fail to override properties that are explicitly set on the command line, as I understand things. Are these include/exclude properties ever explicitly set on the command line (using `-D`)?